src/game/utils/batch/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod report;
```

Comment:
I think we can just move `src/game/batch/report.rs` to `src/game/aggregation.rs`. or `src/game/aggregation/mod.rs` as we discussed before